hack/common.sh

```diff
@@ -266,6 +284,16 @@ sti::build::place_bins() {
   )
 }
 
+# sti::build::make_binary_symlinks makes symlinks for the sti
+# binary in _output/local/go/bin
+sti::build::make_binary_symlinks() {
+  if [[ -f "${STI_LOCAL_BINPATH}/s2i" ]]; then
+    for linkname in "${STI_BINARY_SYMLINKS[@]}"; do
+      ln -sf "${STI_LOCAL_BINPATH}/s2i" "${STI_LOCAL_BINPATH}/${linkname}"
```

Comment:
i don't particularly care but you did it everywhere else so i was surprised.  last i knew linking in windows required third party tooling (though the filesystem supported it).  but you could always make a copy of the binary instead of invoking ln, if it's windows.